Library/Homebrew/dev-cmd/sh.rb

```diff
@@ -10,24 +10,30 @@ module Homebrew
   def sh_args
     Homebrew::CLI::Parser.new do
       usage_banner <<~EOS
-        `sh` [<options>]
+        `sh` [<options>] [<commands>]
 
-        Start a Homebrew build environment shell. Uses our years-battle-hardened
-        Homebrew build logic to help your `./configure && make && make install`
-        or even your `gem install` succeed. Especially handy if you run Homebrew
+        Homebrew build environment that uses years-battle-hardened
+        build logic to help your `./configure && make && make install`
+        and even your `gem install` succeed. Especially handy if you run Homebrew
         in an Xcode-only configuration since it adds tools like `make` to your `PATH`
         which build systems would not find otherwise.
       EOS
       flag   "--env=",
              description: "Use the standard `PATH` instead of superenv's when `std` is passed."
-
-      max_named 0
+      flag   "--cmd=",
```

Comment:
Why is this flag still needed if we're doing `args.named.join` below? I think being consistent with `brew ruby` would be ideal.

Reply:
flag removed. I also got rid of `cmd` variable since it's used in one place only.